ethportal-api/Cargo.toml

```diff
@@ -2,30 +2,27 @@
 name = "ethportal-api"
 version = "0.2.0"
 edition = "2021"
+rust-version = "1.66.1"
```

Comment:
Not sure if we want to add this requirement on a crate that is intended for external use too.